WebRoot/templates/calendar/Appointment.template

```diff
@@ -775,50 +775,54 @@
 				<col style='width:710px' />
 			</colgroup>
 			<tr>
-				<td align='center' id='${id}_navToolbar'>&nbsp;</td>
+				<td align='center' id='${id}_navToolbar' class='ZmSchedulerGridHeaderLeft'><div></div></td>
 				<td valign=bottom>
 					<table role="presentation" width=100% class='ZmSchedulerGridHeaderTable' style='<$= AjxEnv.isIE ? "width:101%" : "" $>'>
 						<tr>
 						<$
-							for (var j = 0; j <= 24; j++) {
-								var hour = AjxDateUtil.isLocale24Hour() ? j : ((j % 12) || 12);
+							var isLocal24Hour = AjxDateUtil.isLocale24Hour();
+							for (var j = 1; j <= 24; j++) {
+								var hour = isLocal24Hour ? j : ((j % 12) || 12);
+								if (!isLocal24Hour) {
+									hour += (j < 12) ? ' AM' : ' PM';
+								}
 						$>
-								<td width=2%><div class='ZmSchedulerGridHeaderCell'><$= hour $></div></td>
+								<td width='4%'><div class='ZmSchedulerGridHeaderCell'><$= hour $></div></td>
 						<$	} $>
 						</tr>
 					</table>
 				</td>
 			</tr>
 		</table>
-		<table role="presentation" class='ZPropertySheet' cellspacing='6'>
+		<table role="presentation" class='ZPropertySheet ZmSchedulerLegendTable' cellspacing='6'>
 			<tr>
 				<td id='${id}_graphKeySpacer'>
 					<span id='${id}_showMoreLink' class="FakeAnchor" style='display:none;'><$= ZmMsg.showMore $></span>
 				</td>
 				<td><div class='ZmGraphKeyColorBox ZmScheduler-working'></div></td>
-				<td class='ZmGraphKeyColorText' style='padding-right:1em;'><$= ZmMsg.free $></td>
+				<td class='ZmGraphKeyColorText' style='padding:0 2em 0 1em;'><$= ZmMsg.free $></td>
```

Comment:
Can we move these style rules to stylesheet?

Reply:
There are style rules for: 
".ZPropertySheet.ZmSchedulerLegendTable>TBODY>TR>TD"
that I want to override in this case, so going for inline styling. 
Let me know if I should change the approach.